pkg/binding/controller.go

```diff
@@ -271,7 +275,9 @@ func (c *Controller) run(ctx context.Context, workers int) error {
 
 	c.logger.Info("Starting workers", "count", workers)
 	for i := 0; i < workers; i++ {
-		go wait.UntilWithContext(ctx, c.runWorker, time.Second)
+		logger := c.logger.WithName(fmt.Sprintf("%s-worker-%d", controllerName, i))
```

Comment:
why not puting worker id in KeyValue pair? 
not very critical but looks cleaner

Reply:
Good idea. Next rev.